src/api-provider.ts

```diff
@@ -99,11 +101,21 @@ export class ApiProvider {
 
     private getTargetprocessConfig() {
         const config = this.store.get(CONFIG_KEYS.TARGETPROCESS);
-
         if (!config) {
             return null;
         }
 
+        // Existing configs won't have subdomain setting, so put it there with existing default.
+        if (!config.subdomain) {
+            config.subdomain = "neworbit";
+            log.info(`Setting TargetProcess to default TP subdomain, '${config.subdomain}'`);
```

Comment:
Should we prompt user to resolve?

Reply:
I think logging what is going on is enough. If we were to stop, and force the user to resolve, it changes this into a breaking change when it doesn't need to be.